boilerplate.typescript.js

```diff
@@ -17,4 +17,5 @@ exports.getPackageTemplates = (packageRelativePath) => [
   { template: 'packages/.yarnrc.ejs', target: `${packageRelativePath}/.yarnrc` },
   { template: 'packages/.npmignore.ejs', target: `${packageRelativePath}/.npmignore` },
   { template: 'packages/package.json.ejs', target: `${packageRelativePath}/package.json` },
+  { template: 'packages/tsconfig.json.ejs', target: `${packageRelativePath}/tsconfig.json` },
```

Comment:
should not be here, there is missing copy command in ignite package function that copy non template files. And it should not have ejs suffix if it's not template.